src/main/java/com/coveros/selenified/services/Call.java

```diff
@@ -34,7 +34,7 @@
  *
  * @author Max Saperstone
  * @version 3.2.0
- * @lastupdate 3/20/2019
+ * @lastupdate 4/4/2019
  */
 public class Call {
     private static final Logger log = Logger.getLogger(Call.class);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "log" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1068)